src/main/java/seedu/address/model/person/Person.java

```diff
@@ -91,6 +91,16 @@ public Set<Tag> getTags() {
         return Collections.unmodifiableSet(tags);
     }
 
+    /**
+     * Checks if the given role is valid.
+     *
+     * @param role The role to check.
+     * @return {@code true} if the role is valid, {@code false} otherwise.
+     */
+    public static boolean isValidRole(String role) {
+        return role.equals("employee") || role.equals("client") || role.equals("supplier");
```

Comment:
could consider using `String#equalsIgnoreCase(...)` since we only accept these three roles that aren't exactly affected by mixed case